.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,68 @@
+name: Build Dockerfile
+on:
+  push:
+    branches: '*'
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/core/cli/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/core/jupyter/docker/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/core/neo4j/docker/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/core/neo4j/importer/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/pipelines/google-poi/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/pipelines/google-poi/src/pipeline/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/pipelines/osm-poi/dockerfile
+    - uses: hadolint/hadolint-action@v1.6.0
+      with:
+        dockerfile: kuwala/pipelines/population-density/dockerfile
+
+    # This is the a separate action that sets up buildx runner
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1 
```

Comment:
main logic as follow:
1. run `hadolint` on all dockerfiles
2. run docker build to make sure no one screw up (there's too many dockerfiles so I only setup 1 docker build in this CI pipeline)

Reply:
yea I've integrated `hadolint` into `pre-commit` hook on my local. but I'm using `npm's pre-commit` 😅 
not sure wether it's ok to commit this `npm's package.json` to this repo lol

Reply:
Hi @zero88 sorry for the late reply, yea make sense about making it output the error and running in parallel!
1. output: current settings would output the error msg [example](https://github.com/david30907d/kuwala/runs/4444786549?check_suite_focus=true)
2. parallel: let me separate them into multiple CI yaml so that they can run in parallel~

Reply:
I use [this pre-commit hook](https://github.com/david30907d/pyproject_template/blob/master/package.json) with hadolint, but as you can see there's many security issues 😅 
need some time to upgrade these packages, or I can create a PR directly since hook are just used on local so no worry about security issue (?)